src/Orleans.TestingHost/TestClusterHostFactory.cs

```diff
@@ -227,12 +227,8 @@ private static void TryConfigureTestClusterMembership(IConfiguration configurati
 
         private static void TryConfigureFileLogging(IConfiguration configuration, IServiceCollection services, string name)
         {
-            bool.TryParse(configuration["UseTestClusterMemebership"], out bool configureFileLogging);
```

Comment:
not sure why turn on file logging has to depend on `UseTestClusterMemebership`. Seems unrelated. so turn it on by default now.

Reply:
I think it was probably a bad copy-paste. this should do `TryParse(configuration[nameof(TestClusterOptions.ConfigureFileLogging)], ...`

---
In reply to: [165226223](https://github.com/dotnet/orleans/pull/3943#discussion_r165226223) [](ancestors = 165226223)